HIP/hip-514-missing-hts-precompile-functions.md

```diff
@@ -0,0 +1,231 @@
+---
+hip: 514
+title: Hedera Token Service Precompiled Contract Token Management Functions
+author: Nana Essilfie-Conduah (@Nana-EC)
+working-group: Danno Ferrin (@shemnon)
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Draft
+created: 2022-06-22
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/516
+updated: 2022-06-28
```

Comment:
status: Last Call
last-call-date-time: 2022-07-26T07:00:00Z
created: 2022-06-22
discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/516
updated: 2022-06-28, 2022-07-15